StortSpelProjekt/Game/src/Gamefiles/Components/AccelerationComponent.cpp

```diff
@@ -25,6 +25,18 @@ void component::AccelerationComponent::Update(double dt)
 	}
 }
 
+void component::AccelerationComponent::OnInitScene()
+{
+}
+
+void component::AccelerationComponent::OnLoadScene()
+{
+}
+
+void component::AccelerationComponent::OnUnloadScene()
+{
+}
```

Comment:
If you write these here with 0 code inside, you should make the functions pure virtual in component.h

Reply:
fixed